### ISSUE TITLE
issue with install script

### ISSUE THREAD

**Issue by andycol** [sipcapture/homer-installer#9]
after running the install script
kamalio fails to start and the error is

0(11390) : <core> [cfg.y:3436]: yyerror_at(): parse error in config file /etc/kamailio/kamailio.cfg, line 44, column 8-16: syntax...

**Comment by lmangani** [comment]
Please include /etc/kamailio/kamailio.cfg, line 44

**Comment by andycol** [comment]
listen=HOMER_LISTEN_PROTO:HOMER_LISTEN_IF:HOMER_LISTEN_PORT

**Comment by lmangani** [comment]
Ok then we need to you review your config block where the substitutions are set. My blind guess is your password uses an illegal char unescaped or something on that line... defaults are:

```
#!substdef "!HOMER_DB_USER!homer_user!g"
#!substdef "!HOMER_DB_PASSWORD!homer_password!g"
#!substdef "!HOMER_LISTEN_PROTO!udp!g"
#!substdef "!HOMER_LISTEN_IF!0.0.0.0!g"
#!substdef "!HOMER_LISTEN_PORT!9060!g"
#!substdef "!HOMER_STATS_SERVER!tcp:HOMER_LISTEN_IF:8888!g"
```

**Comment by andycol** [comment]
i chose the default password when installing

**Comment by andycol** [comment]
if i add the variables manually i now get this error
ERROR: <core> [cfg.y:3299]: yyparse(): cfg. parser: failed to find command report_capture (params 2) 0(12554)...R: <core> [

**Comment by lmangani** [comment]
@andycol its hard if not impossible to troubleshoot without knowing the values you have set

**Comment by andycol** [comment]
@lmangani how can i contact you off here?

**Comment by 00Asgaroth00** [comment]
This issue is corrected in pull request #23
